spec/controllers/canvas_assignment_overrides_controller_spec.rb

```diff
@@ -0,0 +1,99 @@
+require 'rails_helper'
+
+RSpec.describe CanvasAssignmentOverridesController, type: :controller do
+  render_views
+
+  context "with normal signin" do
+    let(:user) { create :admin_user }
+    let(:course) { create :course }
+    let(:section) { create :section, course: course }
```

Comment:
nit: can you do the following to make it clear the intention that this code runs only on cohort schedule sections?

```suggestion
    let(:section) { create :cohort_schedule_section, course: course }
```